src/libs/logos.js

```diff
@@ -63,3 +64,13 @@ export const topBarLogo = () => isTerra() ?
   brandLogoMaker(50, true)
 
 export const footerLogo = () => isTerra() ? terraLogoMaker(terraLogoWhite, { height: 40 }) : brandLogoMaker(40)
+
+export const versionTag = (version, styles) => b({
+  style: {
+    fontSize: 8, lineHeight: '9px',
+    color: 'white', backgroundColor: colors.primary(1.5),
+    padding: '3px 5px', verticalAlign: 'middle',
+    borderRadius: 2, textTransform: 'uppercase',
+    ...styles
+  }
+}, version)
```

Comment:
I think `version` should be in an array since it's expected to be a single string. This could also be upgraded to a true component that takes children to render instead.